crates/pop-common/src/metadata.rs

```diff
@@ -0,0 +1,226 @@
+// SPDX-License-Identifier: GPL-3.0
+
+use scale_info::{form::PortableForm, PortableRegistry, Type, TypeDef, TypeDefPrimitive};
+
+/// Formats a specified type, using the registry to output its full type representation.
+///
+/// # Arguments
+/// * `ty`: A reference to the `Type<PortableForm>` to be formatted.
+/// * `registry`: A reference to the `PortableRegistry` to resolve type dependencies.
```

Comment:
The parameters should be explained. I can see that it is a reference to these types myself :)

Reply:
ty: The type information to format.
registry: The registry used to resolve type dependencies (e.g., subtypes, fields).

Reply:
You are right, not very clear.
Improved here https://github.com/r0gue-io/pop-cli/pull/316/commits/e70bc7a6dbefd8e37165f102654d012605c51a37